www/front_src/src/Resources/columns/index.tsx

```diff
@@ -76,7 +76,7 @@ const ResourcesColumn = ({ Cell, row }: ColumnProps): JSX.Element => {
           <Grid container spacing={1}>
             <Grid item xs={1} />
             <Grid item>
-              <StatusChip statusCode={row.parent.status.code} />
+              <StatusChip statusCode={row.parent?.status?.code || 0} />
```

Comment:
Default status code may be pending

Reply:
Actually this chick should be removed as the API contracts defines parent -> status -> code as mandatory whenever parent is defined